src/storageManager.ts

```diff
@@ -34,6 +34,22 @@ export class StorageManager {
     this.setValue(identity);
   }
 
+  public updateValue(opts: SdkOptions, cookieName: string, previousOpts: SdkOptions) {
+    if (opts.identity) {
+      if (previousOpts.useCookie === true) {
+        this._cookieManager.removeCookie(previousOpts);
+      } else if (!previousOpts || previousOpts.useCookie === false) {
+        this._localStorageManager.removeValue();
+      }
+      this._cookieManager = new CookieManager({ ...opts }, cookieName);
+      this.setValue(opts.identity);
+    }
+  }
+
+  public updateUseCookie(useCookie: boolean) {
```

Comment:
Nitpick: I don't think this is used anymore. Just deleting this function won't mean you need another review.